extensions/phylum.ts

```diff
@@ -4,11 +4,12 @@
 type Package = {
   name: string;
   version: string;
+  package_type: string | undefined;
 };
 
 type Lockfile = {
-  package_type: string;
```

Comment:
This is technically a breaking change. So perhaps we should continue to return something here, even if it's just a placeholder value.

But none of our existing extensions actually use this value as a string. They just pass it back to `analyze` (which now accepts `undefined`). So I am hopeful that we can get away with removing this.

Reply:
If we keep around `package_type` and just add the `format` in addition to the package type, it would still be a breaking change since the type changed, but it would only be an addition and not a change to existing data stored in this struct.

Might be worth it just to not break things? Ultimately we add the information anyway for the `format`.

I guess the problem would be with things like SBOM where the package type cannot be uniquely identified.

Reply:
> it would still be a breaking change since the type changed

That's correct. As far as I can tell, this can't be done without a breaking change.

> I guess the problem would be with things like SBOM where the package type cannot be uniquely identified.

This is exactly what led me to remove `package_type`. Using a value that doesn't match the packages could also break things or just confuse extension authors. So we might as well fix this now, even if it's technically a breaking change.